src/screens/NationDetailsContainer/index.js

```diff
@@ -110,8 +110,9 @@ class NationDetailsContainer extends NavigatorComponent {
   }
 
   performIfHasWallet(functionToPerform) {
+    /* global _ */
```

Comment:
Please insert `import _ from 'lodash'` at the top, rather than using it global.

Reply:
Solved